Assets/Character/AaronTestAssets/PlayerMovement.cs

```diff
@@ -62,7 +67,7 @@ public class PlayerMovement : MonoBehaviour {
                     animator.Play("Idle");
                 }
 
-                if (Input.GetKeyDown(KeyCode.Space))
+                if (Input.GetKeyDown(KeyCode.Space) && downRay.distance < 0.019)
                 {
                     animator.SetBool("OnGround", false);
```

Comment:
Deep nesting of if statements will mess you up later. You can see christian's branch to see how he avoids deep nesting